### PR TITLE
Updated to 0.82

### PR DESCRIPTION
**Pull request by sergiomb2**
While I'm organizing my things a little ... we got this update already for some time, better late than never.

### CODE REVIEW

File: dbgl.spec [hunk 1]
```diff
@@ -123,6 +123,13 @@ appstream-util validate-relax --nonet \
 %{_datadir}/appdata/%{name}.appdata.xml
 
 %changelog
+
+* Sun Nov 05 2017 Unitedrpms Project <unitedrpms AT protonmail DOT com> 0.82-1
```
Comment (xvitaly) [inline dbgl.spec]: Again this s**t repo? Replace this row please by your own.

**Review by xvitaly — CHANGES_REQUESTED** [review]

File: dbgl.spec [hunk 2]
```diff
@@ -123,6 +123,13 @@ appstream-util validate-relax --nonet \
 %{_datadir}/appdata/%{name}.appdata.xml
 
 %changelog
+
```
Comment (igor-raits) [inline dbgl.spec]: no need for empty line here

File: dbgl.spec (hunk 1 above)
Comment (igor-raits) [reply dbgl.spec]: agree with @xvitaly , please use some real person as email.

**Comment by sergiomb2** [comment]
fixed in c93f839

**Review by igor-raits — APPROVED** [review]
lgtm

**Review by xvitaly — APPROVED** [review]

**Pull request merged**